src/common/telemetry.py

```diff
@@ -0,0 +1,32 @@
+from applicationinsights import TelemetryClient
+from .telemetry_events import TelemetryEvent
+
+
+class Telemetry:
+    def __init__(self):
+        # State of the telemetry
+        self.__enable_telemetry = True
+        self.telemetry_client = TelemetryClient("__AIKEY__")
```

Comment:
Just curious, how does this part work? How to you get the key?

Reply:
it is replaced in the release pipeline for when we create the vsix